cmd/influx/write.go

```diff
@@ -89,6 +91,7 @@ func cmdWrite(f *globalFlags, opt genericCLIOpts) *cobra.Command {
 	cmd.PersistentFlags().MarkHidden("xIgnoreDataTypeInColumnName") // should be used only upon explicit advice
 	cmd.PersistentFlags().StringVar(&writeFlags.Encoding, "encoding", "UTF-8", "Character encoding of input files or stdin")
 	cmd.PersistentFlags().StringVar(&writeFlags.ErrorsFile, "errors-file", "", "The path to the file to write rejected rows to")
+	cmd.PersistentFlags().Float64Var(&writeFlags.LimitRate, "limit-rate", 0.0, "How many megabytes per minute the write will allow. Defaults to zero, which disables throttling.")
```

Comment:
I'm inclined to call this `rate-limit`, which is more conventional 

```suggestion
	cmd.PersistentFlags().Float64Var(&writeFlags.LimitRate, "rate-limit", 0.0, "Enable rate limiting at the specified megabytes per minute. Defaults to 0, which disables rate limiting")
```

Reply:
I agree with @glinton suggestion, a KB/s is a better metric